thrall/app/lib/kinesis/ThrallEventConsumer.scala

```diff
@@ -58,15 +58,21 @@ class ThrallEventConsumer(es: ElasticSearch6,
                 case e: Throwable =>
                   Logger.error(s"Failed to process ${updateMessage.subject} message; message will be ignored:", e)(updateMessage.toLogMarker)
               }
-
-              Await.ready(eventuallyAppliedUpdate, ThirtySeconds)
+              try {
+                Await.ready(eventuallyAppliedUpdate, ThirtySeconds)
+              } catch {
+                case e: TimeoutException =>
+                  Logger.error(
+                    s"Timeout while processing ${updateMessage.subject} message; message will be ignored:",
```

Comment:
minor: Could we include the duration we waiting for in the log too to save us remembering/looking up what the value is?

(`ThirtySeconds` is a badly named variable, `Timeout` is more descriptive imo)

Reply:
yeah, that's sensible

Reply:
good catch